lib/src/over_react_redux/over_react_flux.dart

```diff
@@ -0,0 +1,503 @@
+///     state = ReduxState('default state');
+///
+///     triggerOnActionV2(_actions.fluxAction,
+///         () => this.influxReducer(ReduxAction('New Value')));
+///   }
+///
+/// }
+///
+/// class ReduxAction {
+///   String value;
+///
+///   ReduxAction(this.value);
+/// }
+///
+/// class ReduxState {
+///   String valueFromState;
+///
+///   ReduxState(this.valueFromState);
+/// }
+///
+/// ReduxState exampleReducer(ReduxState oldState, Object action) {
+///   if (action is ReduxAction) return ReduxState(action.value);
+///
+///   return oldState;
+/// }
+/// ```
+mixin InfluxStoreMixin<S> on flux.Store {
+  /// A traditional Redux reducer function that should return a new instance of
+  /// the corresponding state class.
+  ///
+  /// This is used in [influxReducer] to update the [state] field before triggering
+  /// an update.
+  redux.Reducer<S> get reduxReducer;
+
+  /// An instance of the Redux state model that the Flux store is migrating to.
+  S state;
+
+  /// A field to track if Flux has already tried to update Redux.
+  bool _isReduxInSync = false;
+
+  /// Checks to see if the Redux store needs to be updated and triggers an action
+  /// if so.
+  ///
+  /// This is only to be used from within the [FluxToReduxAdapterStore] to control
+  /// when Redux needs to receive an update to keep it in sync with Flux.
+  void triggerReduxUpdateFromFlux(Dispatcher dispatcher) {
+    // `state` can be null if `ConnectableFluxStore` is being used.
+    if (_isReduxInSync && state != null) {
+      _isReduxInSync = false;
+      return;
+    }
+
+    dispatcher(_FluxStoreUpdatedAction());
+  }
+
+  /// A wrapper around a pure Redux reducer that keeps the Flux UI up to date with
+  /// store changes.
+  void influxReducer(dynamic action) {
+    if (action is _FluxStoreUpdatedAction) {
+      _isReduxInSync = true;
+      return;
+    }
+
+    final oldState = this.state;
+    this.state = reduxReducer(this.state, action);
+
+    // If Redux has mutated the store, we need to keep Flux in sync
+    if (oldState != this.state) this.trigger();
+  }
+}
+
+/// A class that can be used to make a [flux.Store] compatible with [connectFlux]
+/// without adding the Redux implementation.
+///
+/// The store should still be wrapped with an adapter store, with [Null] being
+/// the generic for the Redux state class. Then the usage is the same as [connectFlux]
+/// with a standard [FluxToReduxAdapterStore].
+///
+/// __Note:__ Naturally because this adapter store does not have a real Redux reducer,
+/// a `connected` component cannot trigger actions. If the library in the position to
+/// have Redux components, this class should not be used and the Redux boilerplate
+/// should be added. If needed, however, a `connect`ed component can receive updates from
+/// this implementation (just not trigger them).
+///
+/// __Example:__
+/// ```dart
+/// import 'package:w_flux/w_flux.dart' as flux;
+///
+/// class ExampleFluxStore extends ConnectableFluxStore {
```

Comment:
Instead of requiring both a parent class and a wrapper, could we make a wrapper store that accepts any kind of flux store, like it did originally?
```dart
class FluxToReduxAdapterStoreWithoutInflux<S extends flux.Store, V> extends redux.Store<S> {
  final S store;

  StreamSubscription _storeListener;

  FluxToReduxAdapterStoreWithoutInflux(this.store, dynamic actions, {List<redux.Middleware<S>> middleware})
      : super((_, __) => store, middleware: middleware ?? const [], initialState: store, distinct: false) {
    assert(store is! InfluxStoreMixin, 'Use FluxToReduxAdapterStore when your store implements InfluxStoreMixin');
    
    _storeListener = store.listen((_) {
      dispatch(_FluxStoreUpdatedAction());
    });

    actionsForStore[store] = actions;
  }

  @override
  Future teardown() async {
    await _storeListener.cancel();
    await super.teardown();
  }
}
```

Reply:
Omg just had an idea: we use extension methods here or other places to make the boilerplate better (since I know that was a concern). Just an idea though.
```dart
extension<S extends InfluxStoreMixin> on S {
  FluxToReduxAdapterStore<S, Null> asReduxStore(dynamic actions, {List<redux.Middleware<S>> middleware}) {
    return FluxToReduxAdapterStore(this, actions, middleware: middleware);
  }
}
extension<S extends flux.Store> on S {
  FluxToReduxAdapterStoreWithoutInflux<S, Null> asReduxStoreWithoutInflux(dynamic actions, {List<redux.Middleware<S>> middleware}) {
    return FluxToReduxAdapterStoreWithoutInflux(this, actions, middleware: middleware);
  }
}
```
```diff
 var actions = ExampleActions();
 var store = ExampleStore(actions);

-var adapterStore = FluxToReduxAdapterStore(store, actions);
+var adapterStore = store.asReduxStore(actions);
```

Reply:
Ohhh that's dope and a great idea!

Reply:
@davidmarne-wf let me know if you see any downsides to the approach over the `ConnectableFluxStore`, but it seems to accomplish the same goal!

Reply:
this looks legit to me, i like it